.github/workflows/unit_tests.yaml

```diff
@@ -130,14 +130,16 @@ jobs:
           sudo apt-get -y update
           # Needed for unit tests
           sudo apt-get -y install \
-              coreutils cvs gfortran graphviz gnupg2 mercurial  ninja-build \
+              coreutils cvs gfortran graphviz gnupg2 mercurial ninja-build \
               patchelf
           # Needed for kcov
           sudo apt-get -y install cmake binutils-dev libcurl4-openssl-dev
           sudo apt-get -y install zlib1g-dev libdw-dev libiberty-dev
     - name: Install Python packages
       run: |
           pip install --upgrade pip six setuptools codecov coverage[toml]
+          # ensure style checks are not skipped in unit tests for python >= 3.6
+          python2 -c 'import sys; sys.exit(sys.version_info < (3, 6))' && pip install --upgrade flake8 isort>=4.3.5 mypy>=0.900 black
```

Comment:
Why `python2`, won't this always be true even if we are running on Python 3 for the other tests?

Reply:
And should the comment be updated (it references python >= 3.6).

Reply:
> And should the comment be updated (it references python >= 3.6).

It's correct -- the installs are run when python >= 3.6

Reply:
Fixed -- this was a copy-paste error

Reply:
Ah that answers my question toot then :)